docs/runbooks/build_new_mitx_env.md

```diff
@@ -0,0 +1,32 @@
+In order to provision a new MITx environment the following steps are necessary:
+
+- Define the environment in the `environments.yml` file in the pillar repository
+- Provision the VPC
+  `VPC_NAME='My VPC' ENVIRONMENT=my-vpc BUSINESS_UNIT=residential salt-run state.orchestrate orchestrate.aws.mitx`
+- Accept the VPC peering connection via the AWS console
+- Update the route tables in the operations VPC to allow traffic from the new environment
+- Update the operations security groups for Consul and FluentD to allow traffic from the new environment
```

Comment:
Are there issues created for those manual steps above to possibly automate them?